inventories/broker.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import yaml
+import os
+import subprocess
+import sys
+
+try:
+    from StringIO import StringIO  # pyright: reportMissingImports=false
+except ImportError:
+    from io import StringIO  # pyright: reportMissingImports=false
```

Comment:
If you make it Python 3-only you can also simplify this to just `from io import StringIO`.